templates/serverpod_templates/projectname_server/lib/src/endpoints/example_endpoint.dart

```diff
@@ -16,6 +16,12 @@ class ExampleEndpoint extends Endpoint {
   // supported. The `session` object provides access to the database, logging,
   // passwords, and information about the request being made to the server.
   Future<String> hello(Session session, String name) async {
+    return ExampleUtil.buildGreeting(name);
+  }
+}
+
+abstract final class ExampleUtil {
+  static String buildGreeting(String name) {
     return 'Hello $name';
   }
 }
```

Comment:
Discussion: Should we have this in a lib file instead? That is probably the project structure we would like to suggest. Not importing the endpoint in the unit test?

Reply:
Yes, it does make sense! However, I was weighing it against creating too many example files. We don't want too many generated files which can feel like a lot of "trash". What do you think about this aspect?

Reply:
Alright, I'll add that then 👍